index.html

```diff
@@ -1,41 +1,285 @@
+                                <li class="list-item skills-item">Html</li>
+                                <li class="list-item skills-item">CSS</li>
+                            </ul>
+                        </div>
+                    </section>
+                </div>
+
+                <div class="card about-card">
+                    <img class="card-float-img" src="./static/images/frameworks-icon.png" alt="Language logo">
+                    <section class="card-body skills-body">
+                        <h5>
+                            Frameworks
+                        </h5>
+                        <div class="work-card-list">
+                            <ul class="list-view">
+                                <li class="list-item skills-item">Bootstrap</li>
+                                <li class="list-item skills-item">Ruby on rails</li>
+                                <li class="list-item skills-item">RSpec</li>
+                                <li class="list-item skills-item">Capybara</li>
+                                <li class="list-item skills-item">Selenium</li>
+                            </ul>
+                        </div>
+                    </section>
                 </div>
-                <!-- social links -->
+
+                <div class="card about-card">
+                    <img class="card-float-img" src="./static/images/skills-icon.png" alt="Language logo">
+                    <section class="card-body skills-body">
+                        <h5>
+                            Skills
+                        </h5>
+                        <div class="work-card-list">
+                            <ul class="list-view">
+                                <li class="list-item skills-item">Codekit</li>
+                                <li class="list-item skills-item">GitHub</li>
+                                <li class="list-item skills-item">Codepen</li>
+                                <li class="list-item skills-item">Gitlab</li>
+                                <li class="list-item skills-item">Terminal</li>
+                            </ul>
+                        </div>
+                    </section>
+                </div>
+            </div>
+        </section>
+
+        <!-- contact section -->
+        <section class="main-l-item contact-container">
+            <form class="contact-form" action="https://formspree.io/f/mayvlerd" method="post">
+                <h3>
+                    I'm always interested in hearing about new projects,
+                    so if you'd like to chat please get in touch.
+                </h3>
+                <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
+                <input type="email" id="mail" name="mail" placeholder="Email address" required>
+                <textarea id="msg" name="msg" placeholder="Enter text here" rows="4" maxlength="500"></textarea>
```

Comment:
In order for a user to be impossible to send an empty form, we need validation on `textarea` too.

- [ ] Kindly, implement HTML validation on `textarea`;